scripts/lease-email.py

```diff
@@ -0,0 +1,97 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import argparse
+import datetime
+import email.message
+import json
+import pathlib
+import smtplib
+
+import openstack
+
+import esi
+
+
+def send_email(from_email, to_email, subject, body,
+               smtp_server, smtp_port, smtp_username, smtp_password):
+    msg = email.message.EmailMessage()
+    msg.set_content(body)
+    msg['From'] = from_email
+    msg['To'] = to_email
+    msg['Subject'] = subject
+
+    with smtplib.SMTP(smtp_server, smtp_port) as mailer:
+        mailer.starttls()
+        mailer.login(smtp_username, smtp_password)
+        mailer.send_message(msg)
```

Comment:
This will only work for our SMTP server which is okay for our use case. If you wanted to make the script more generic we would need config options for:

* Connection type (Basic vs TLS vs STARTTLS)
* Whether or not smtp creds are required (not all SMTP servers require this, especially internal ones)

Reply:
I'd prefer to punt that functionality to later, if it's ever needed.